_markbind/layouts/default.md

```diff
@@ -66,6 +76,14 @@
   * [Suspense]({{baseUrl}}/docs/built-ins/suspense.html)
 * Scaling Up
 * Best Practices
```

Comment:
Perhaps you could add ` :expanded:` after `Best Practices` to keep the dropdown expanded